### DISCUSSION TITLE
windows trash folder

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#2063] [Ideas]
In the Windows $recyclebin folder (trash folder), the deleted files have a different name, which usually starts with $R, for example: $R4QVF06.mp4 with their respective metadata, which are not the same as the file that was deleted.
For each one there is another file that starts with the letter I, in the example $I4QVF06.mp4. Inside it, there is the name of the original file and the full path with the metadata.

Wouldn't it be interesting to have this information highlighted elsewhere, in the headings of the Windows Recycle Bin category?

paulo

**Comment by lfcnassif** [comment]
Hi @paulobreim. Not sure what you are asking for... If you click on $I.... artifacts, the decoded information is being shown, right?

**Reply by paulobreim** [reply]
Yes, if I click on $Ixx I can see at the bottom, on the right, by clicking in text the original path of the file.
But I didn't see the metadata part (date created, modified, etc) of the original file.
So I thought that to have a quick overview of everything that is in the trash, the original name and other information could be in the columns.
